### PR TITLE
Variety Up & Meeting PPL

### PR DESCRIPTION
**Pull request by Eiron**
This is a slight modification to increase variety on jukebox, whilst ensuring that the updated terms of our PPL license are adhered to.

### CODE REVIEW

File: src/Classes/ServiceAPI/MyRadio_TracklistItem.php [hunk 1]
```diff
@@ -404,7 +405,7 @@ public static function getIfAlbumArtistCompliant(MyRadio_Track $track, $include_
             }
         }
 
-        return $result[0] < 2;
+        return $result[0] = 0;
```
Comment (LordAro) [inline src/Classes/ServiceAPI/MyRadio_TracklistItem.php]: eep.

Comment (Eiron) [inline src/Classes/ServiceAPI/MyRadio_TracklistItem.php]: <1 maybe? All I know is that it should only return a result if there are no matches within an hour.

Comment (Eiron) [inline src/Classes/ServiceAPI/MyRadio_TracklistItem.php]: Oh. ==0. I code good.

**Comment by LordAro** [comment]
Note: Merging this will require pulling myradio on dolby (unless someone gets around to removing that dependency before then)

**Comment by Eiron** [comment]
The error that prompted this change may still be lurking somewhere, but this will ensure we are sticking more stringently to the terms of the license. It will certainly be more obvious if something is not working right with this code in play.

**Pull request merged**